server/pkg/weather/weather.go

```diff
@@ -0,0 +1,150 @@
+package weather
+
+import "time"
+
+// DayOrNight returns friendly name dependent on the time for which the
+// sun sets or rises in relation to the current time
+func DayOrNight(current, rise, set time.Time) string {
+	if rise.Before(current) && current.Before(set) {
```

Comment:
```suggestion
func DayOrNight(current, sunRise, sunSet time.Time) string {
	if sunRise.Before(current) && current.Before(sunSet) {
```
Probably want to rename these since `set` could be confused with something else.